streams-contrib/streams-persist-graph/src/main/java/org/apache/streams/graph/GraphHttpPersistWriter.java

```diff
@@ -218,7 +217,7 @@ protected ObjectNode executePost(HttpPost httpPost) {
     @Override
     public void prepare(Object configurationObject) {
```

Comment:
but in the event the method caller *does* pass in an object, do you want to try to do something with it?